tests/src/Integration/Action/EntityPathAliasCreateTest.php

```diff
@@ -0,0 +1,144 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\Tests\rules\Integration\Action\EntityPathAliasCreateTest.
+ */
+
+namespace Drupal\Tests\rules\Integration\Action;
+
+use Drupal\Tests\rules\Integration\RulesEntityIntegrationTestBase;
+
+/**
+ * @coversDefaultClass \Drupal\rules\Plugin\Action\EntityPathAliasCreate
+ * @group rules_actions
+ */
+class EntityPathAliasCreateTest extends RulesEntityIntegrationTestBase {
+
+  /**
+   * The action to be tested.
+   *
+   * @var \Drupal\rules\Core\RulesActionInterface
+   */
+  protected $action;
+
+  /**
+   * The mocked alias storage service.
+   *
+   * @var \PHPUnit_Framework_MockObject_MockObject|\Drupal\Core\Path\AliasStorageInterface
+   */
+  protected $aliasStorage;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function setUp() {
+    parent::setUp();
+
+    // Prepare mocked AliasStorageInterface.
+    $this->aliasStorage = $this->getMockBuilder('Drupal\Core\Path\AliasStorageInterface')
+      ->setMethods(['save'])
+      ->getMockForAbstractClass();
+
+    $this->container->set('path.alias_storage', $this->aliasStorage);
+
+    // Instantiate the action we are testing.
+    $this->action = $this->actionManager->createInstance('rules_entity_path_alias_create:entity:entity_test');
+  }
+
+  /**
+   * Tests the summary.
+   *
+   * @covers ::summary
+   */
+  public function testSummary() {
+    $this->assertEquals('Create entity_test path alias', $this->action->summary());
+  }
+
+  /**
+   * Tests the action execution with an unsaved node.
```

Comment:
I thought there were 2 instances, but it's only one. Instead of "Tests the action execution with an unsaved node" it should be "Tests the action execution with an unsaved entity".
This is in tests/src/Integration/Action/EntityPathAliasCreateTest.php at line 59